api/v1/storagecluster_types.go

```diff
@@ -456,18 +456,33 @@ type EncryptionSpec struct {
 	Enable bool `json:"enable,omitempty"`
 	// +optional
 	ClusterWide bool `json:"clusterWide,omitempty"`
+	// Configure the default rbd encrypted storage class
 	// +optional
 	StorageClass bool `json:"storageClass,omitempty"`
 	// StorageClassName specifies the name of the storage class created for ceph encrypted block pools
 	// +kubebuilder:validation:MaxLength=253
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
-	StorageClassName     string                   `json:"storageClassName,omitempty"`
+	StorageClassName string `json:"storageClassName,omitempty"`
+	// Configure the default CephFS encrypted storage class
+	// +optional
+	CephFs               DefaultStorageClassSpec  `json:"cephfs,omitempty"`
 	KeyManagementService KeyManagementServiceSpec `json:"kms,omitempty"`
 	// KeyRotation defines options for Key Rotation.
 	// +optional
 	KeyRotation KeyRotationSpec `json:"keyRotation,omitempty"`
 }
 
+type DefaultStorageClassSpec struct {
```

Comment:
```suggestion
type StorageClassSpec struct {
}
// or 
type CSIStorageClassSpec struct {
}
```

Reply:
default doesnt suit as its a shared struct for cephfs/rbd/nfs

Reply:
Changed it to StorageClassSpec